### ISSUE TITLE
Question about fine-tuning

### ISSUE THREAD

**Issue by L2531-M** [materialsvirtuallab/matgl#642]
I'm currently trying to fine-tune several matgl models, but I noticed that the accuracy of the fine-tuned models is actually lower than that of the pre-trained models.
Here are the training results. I would really appreciate it if someone could take a look and let me know if I might be making a mistake.
# my consequence
(model1=CHGNet-MatPES-r2SCAN-2025.2.10-2.7M-PES  model2=M3GNet-MatPES-r2SCAN-v2025.1-PES)

**Energy plot before training:**
Slope model 1: 0.9275
Slope model 2: 0.9552
MAE model1: 0.0093
MAE model2: 0.0101

**Energy plot after training:**
Slope model1: 0.2016
Slope model2: 0.4355
MAE model1: 0.0223
MAE model2: 0.0189

The plot of the results is attached below.
[training_consequence.pdf](https://github.com/user-attachments/files/22017150/training_consequence.pdf)

# my training code

    from __future__ import annotations
    
    import os
    import shutil
    import warnings
    from functools import partial
    
    import lightning as pl
    import numpy as np
    from dgl.data.utils import split_dataset
    from pytorch_lightning.loggers import CSVLogger
    
    import matgl
    from matgl.config import DEFAULT_ELEMENTS
    from matgl.ext.pymatgen import Structure2Graph
    from matgl.graph.data import MGLDataLoader, MGLDataset, collate_fn_pes
    from matgl.models import CHGNet
    from matgl.utils.training import PotentialLightningModule
    
    # To suppress warnings for clearer output
    warnings.simplefilter("ignore")
    
    
    import os
    from pymatgen.io.vasp import Vasprun
    
    structure_list = []
    energy_list = []
    force_list = []
    stress_list = []
    
    # data
    base_dir = "Data_1"
    vasp_dirs = [os.path.join(base_dir, str(i)) for i in range(0, 300)]
    
    for vasp_dir in vasp_dirs:
        vasprun_path = os.path.join(vasp_dir, "vasprun.xml")
    
        try:
            vr = Vasprun(vasprun_path, parse_potcar_file=False)
    
            structure = vr.final_structure
            energy = vr.final_energy
            force = vr.ionic_steps[-1]["forces"]
    
            structure_list.append(structure)
            energy_list.append(energy)
            force_list.append(force)
    
            print(f"[OK] Loaded {vasprun_path}")
    
        except FileNotFoundError:
            print(f"[SKIP] Not found: {vasprun_path}")
        except Exception as e:
            print(f"[ERROR] Failed to load {vasprun_path}: {e}")
    
    
    labels = {
        "energies": [float(e) for e in energy_list],
        "forces": [f.tolist() for f in force_list]
    }
    
    element_types = DEFAULT_ELEMENTS
    converter = Structure2Graph(
        element_types=["Li", "V", "Nb", "O"],
        cutoff=4.0,
    )
    
    dataset = MGLDataset(
        threebody_cutoff=3,
        structures=structure_list,
        converter=converter,
        labels=labels,
        include_line_graph=True,
    )
    
    
    train_data, val_data, test_data = split_dataset(
        dataset,
        frac_list=[0.8, 0.1, 0.1],
        shuffle=True,
        random_state=42,
    )
    # if you are not intended to use stress for training, switch include_stress=False!
    my_collate_fn = partial(collate_fn_pes, include_line_graph=True, include_stress=False)
    
    train_loader, val_loader, test_loader = MGLDataLoader(
        train_data=train_data,
        val_data=val_data,
        test_data=test_data,
        collate_fn=my_collate_fn,
        batch_size=1,
        num_workers=0,
    )
    
    
    # download a pre-trained M3GNet
    chgnet_nnp = matgl.load_model("CHGNet-MatPES-PBE-2025.2.10-2.7M-PES")
    model_pretrained = chgnet_nnp.model
    # obtain element energy offset
    property_offset = chgnet_nnp.element_refs.property_offset
    # you should test whether including the original property_offset helps improve training and validation accuracy
    lit_module_finetune = PotentialLightningModule(
        model=model_pretrained, element_refs=property_offset, lr=1e-5, include_line_graph=True
    )
    
    # If you wish to disable GPU or MPS (M1 mac) training, use the accelerator="cpu" kwarg.
    logger = CSVLogger("logs", name="CHGNet_finetuning")
    trainer = pl.Trainer(max_epochs=100, accelerator="cuda", logger=logger, inference_mode=False)
    trainer.fit(model=lit_module_finetune, train_dataloaders=train_loader, val_dataloaders=val_loader)
